src/tools/CrosshairsTool.js

```diff
@@ -189,3 +192,24 @@ export default class CrosshairsTool extends BaseTool {
     });
   }
 }
+
+const crosshairsCursor = {
+  svgGroupString: `<path fill="ACTIVE_COLOR" d="M1325 1024h-109q-26 0-45-19t-19-45v-128q0-26
```

Comment:
Instead of keeping the SVG path data in their respective tools, can we put them somewhere co-located that we can import from? It might make it easier for tools to share cursors; especially 3rd party ones (or to add to the list of usable cursors).

Reply:
Yeah this makes sense. We should probably also instantiate them in that file, and pass an actual `MouseCursor` instance to the constructor of `BaseTool`, rather than passing a string and then building the `MouseCursor` inside.

Then to override the cursor you pass a new `MouseCursor` when instantiating it. Same for third-party tools.